FeedAPIChallenge/Feed API/FeedImagesMapper.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  FeedImagesMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Romeo Flauta on 4/10/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal final class FeedImagesMapper {
+	private struct Root: Decodable {
+		let items: [Item]
+		var feed: [FeedImage] {
+			return items.map { $0.item }
+		}
+	}
+
+	//internal representation of FeedImage but for the API module
+	private struct Item: Decodable {
+		let imageId: String
```

Comment:
The `imageId` should be of type `UUID` since the `UUID` type conforms to `Decodable` and can be decoded like `String`, `URL`, etc..

```suggestion
		let imageId: UUID
```